metrics/web/docs/01_metrics_web_application_architecture.md

```diff
@@ -71,7 +71,7 @@ Currently, our application has the following modules:
 
 Let us consider the class diagram that will explain the main idea of the package structure and relationships between application layers in one module. Let's call this module `cool_module`: 
 
-![Web Architecture Class Diagram](http://www.plantuml.com/plantuml/proxy?cache=no&fmt=svg&src=https://raw.githubusercontent.com/software-platform/monorepo/master/metrics/web/docs/diagrams/web_architecture_class_diagram.puml)
+![Web Architecture Class Diagram](http://www.plantuml.com/plantuml/proxy?cache=no&fmt=svg&src=https://raw.githubusercontent.com/software-platform/monorepo/web_app_docs_improvements/metrics/web/docs/diagrams/web_architecture_class_diagram.puml)
```

Comment:
Should we also change namings in this digram? CoolViewModel -> CoolTileViewModel?